NuGet.Config

```diff
@@ -6,5 +6,6 @@
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="myget.org dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
```

Comment:
@brthor - is this still needed?  I wasn't sure.  I needed it for the MSBuild nuget packages I was using in order to make our classes inherit from ITask.